activerecord/test/cases/migration/command_recorder_test.rb

```diff
@@ -460,6 +460,11 @@ def test_invert_transaction_with_irreversible_inside_is_irreversible
         end
       end
 
+      def test_invert_add_check_constraint
```

Comment:
Maybe split this into 2 test cases, like `test_invert_remove_check_constraint` and `test_invert_remove_check_constraint_if_exists`?